code/model/ForumThread.php

```diff
@@ -133,7 +133,7 @@ function getFirstPost() {
 	 * @return int
 	 */
 	function getNumPosts() {
-		return (int)DB::query("SELECT count(*) FROM \"Post\" WHERE \"ThreadID\" = $this->ID")->value();
+		return (int)DB::query(sprintf('SELECT COUNT(Post.ID) FROM Post LEFT JOIN Member ON Post.AuthorID = Member.ID WHERE ThreadID = %s AND Member.ForumStatus = \'Normal\'', $this->ID))->value();
```

Comment:
120 chars, fair call.

We used to have a lot of forum implemented in ORM however found we had performance issues. Subsequently a lot ended up getting reworked into SQL calls when we relaunched the ssilverstripe.org site last year. 

Any thoughts on ORM or SQL given speed concerns?